app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php

```diff
@@ -318,12 +318,20 @@ protected function populateSelectionTemplate($selection, $optionId, $parentId, $
         return $populatedSelection;
     }
 
+    /**
+     * @deprecated Misspelled method - use retrieveProductsByCachedSkus() instead
```

Comment:
Please use ```@see retrieveProductsByCachedSkus``` instead of description in ```@deprecated```

Reply:
Updated in d3574d3. Is it ok to leave the short description to `@deprecated` ?

Reply:
Sure